packages/gateway/src/features/transformation/index.ts

```diff
@@ -1,3 +1,4 @@
+import { IAuthHeader } from '@gateway/common-types'
```

Comment:
This change improves the error messaging we get when sending invalid data to declare/register/certify handlers

If I for instance send Mother's `dateOfMarriage` fields as `null`, I previously got:

```
Expected undefined to be a FieldBuilder object for field name dateOfMarriage. The current field value is null.
```

as some payloads might have multiple `dateOfMarriage` fields, it was sometimes difficult to see from which it originated. Now the error message would be:

This change improves the error messaging we get when sending invalid data to declare/register/certify handlers

If I for instance send Mother's `dateOfMarriage` fields as `null`, I previously got:

```
Expected undefined to be a FieldBuilder object for field name mother.dateOfMarriage. The current field value is null.
```

Reply:
Nice! This will save debugging time related to invalid declaration data.